dcgm/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - DCGM
+
+## 1.0.0 / 2023-05-17
+
+* [Added] Add the DCGM integration. See [#](https://github.com/DataDog/integrations-core/pull/[****]).
```

Comment:
Could you remove that part, ddev automatically takes care of the changelog when we release an integration. More info [here](https://datadoghq.dev/integrations-core/guidelines/pr/#title)